packages/autocomplete/Readme.md

```diff
@@ -198,6 +198,35 @@ The prop `multipleSelect` can be used to allow multi selecting items.
 </Autocomplete>
 ```
 
+The prop `selection` can be used to show selected items when the component gets loaded.
+```
+<Autocomplete
+  items={[
+    { label: "Antwerpen", value: "antwerpen"},
+    { label: "Gent", value: "gent"},
+    { label: "Brussel", value: "brussel"},
+    { label: "Brugge", value: "brugge"},
+    { label: "Hasselt", value: "hasselt"},
+    { label: "Luik", value: "luik"},
+    { label: "Oostende", value: "oostende"},
+    { label: "Namen", value: "namen"},
+    { label: "Mechelen", value: "mechelen"},
+    { label: "Sint-Niklaas", value: "sint-niklaas"},
+    { label: "Aalst", value: "aalst"},
+    { label: "Genk", value: "genk"}
+  ]}
+  selection={[
```

Comment:
@MaartenDeWeerdt  I don't know the context of this PR, but in accordance with preventing data duplication, it seems to me that it would be better to have an optional property on the items, such as 'defaultSelected'; that a use can set to true. This prevents the fact that data has to be replicated. Thank you. @TriangleJuice

Reply:
I don't think that's a good thing too do. This component supports lazy loading, so it is possible that the selected data isn't available yet. 

The context is simple, we need to set a default value.

```javascript
// use the users data to set the current selection
const formValue = {
    users: [{
       id: '1234',
       name: 'john doe'
    }]
}

// render a form with a lazy loaded autocomplete field that searches for users in the system
// and set the default value 
...
```

Reply:
@glennverschooren The solution of copying an array of items that will be default selected will only work for sync items, and will not take into account asynchronous items. Setting a default value for this control should work for both cases. There are already classes that handle the sync/async states. 

I suggest to add a property on the items (defaultSelected) , that can be read from synchronous items as async items, initial selection should then be set based on that property.

Reply:
I have opend #151 which uses the existing `defaultValue` property